Doc/library/io.rst

```diff
@@ -401,29 +401,29 @@ I/O Base Classes
       hint.
 
       Note that it's already possible to iterate on file objects using ``for
-      line in file: ...`` without calling ``file.readlines()``.
+      line in file: ...`` without calling :meth:`!file.readlines`.
 
    .. method:: seek(offset, whence=SEEK_SET, /)
 
       Change the stream position to the given byte *offset*.  *offset* is
       interpreted relative to the position indicated by *whence*.  The default
-      value for *whence* is :data:`SEEK_SET`.  Values for *whence* are:
+      value for *whence* is :data:`!SEEK_SET`.  Values for *whence* are:
 
-      * :data:`SEEK_SET` or ``0`` -- start of the stream (the default);
+      * :data:`!SEEK_SET` or ``0`` -- start of the stream (the default);
         *offset* should be zero or positive
-      * :data:`SEEK_CUR` or ``1`` -- current stream position; *offset* may
+      * :data:`!SEEK_CUR` or ``1`` -- current stream position; *offset* may
         be negative
-      * :data:`SEEK_END` or ``2`` -- end of the stream; *offset* is usually
+      * :data:`!SEEK_END` or ``2`` -- end of the stream; *offset* is usually
         negative
 
       Return the new absolute position.
 
       .. versionadded:: 3.1
-         The ``SEEK_*`` constants.
+         The :data:`!SEEK_*` constants.
```

Comment:
I think this should stay as literal markup; SEEK_* isn't a valid identifier:

```suggestion
         The ``SEEK_*`` constants.
```

Reply:
I think it should be marked up correctly, nevertheless. We do simular tricks in other parts of the docs.